src/motherduck_destination_server.cpp

```diff
@@ -584,7 +594,8 @@ ::grpc::Status DestinationSdkImpl::WriteHistoryBatch(
     for (auto &filename : request->replace_files()) {
       logger->info("replace/upsert file " + filename);
       IngestProperties props =
-          create_ingest_props(filename, request, column_names, csv_block_size);
+          create_ingest_props(filename, request, cols, csv_block_size,
+                              UnmodifiedMarker::Disallowed);
```

Comment:
same as in the other batch logic - upsert files can contain strings representing null values